save-backend/src/main/resources/application.properties

```diff
@@ -1,4 +1,4 @@
 server.port = 5000
 management.endpoints.web.exposure.include=health,info,prometheus
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5Dialect
-spring.liquibase.enabled=false
\ No newline at end of file
+spring.profiles.active=dev
```

Comment:
Why? Should we set profile here at all? I believe for tests we have annotation, and for prod an argument should be added to launch command (in `SpringBootCnfiuration`, I guess).